viz_config.yml

```diff
@@ -21,6 +21,7 @@ shift:
 
 # fetch site data parameters
 request_limit: 10
+request_limit_historic: 5
```

Comment:
Is 5 really the most sites you can reasonably request at a time? that's a really smaller number = lot of dataRetrieval calls!

Reply:
Well, I was looking at what we did for the nwis part of the nawqa pulls (https://github.com/USGS-R/nawqa_wqp/blob/master/1_nwisdata/cfg/nwis_pull.yml) which was 250,000 and I calculated ~ how many sites that would be if all the sites in a call had the max amt of data (250,000 / (365*150) = ~5)

Reply:
Nice to hear (and not surprising) that this was a well-reasoned decision. And saw the good news on Slack that this only took 6 hours :+1: